website/content/docs/connect/manage-traffic/route-to-local-upstreams.mdx

```diff
@@ -10,9 +10,9 @@ This topic describes how to enable locality-aware routing so that Consul can pri
 
 <EnterpriseAlert> This feature is available in Consul Enterprise. </EnterpriseAlert>
 
-!> **Warning**: Locality-aware routing is an advanced feature that may adversely impact service capacity if used incorrectly. By default, this feature routes 100% of traffic to the most local set of service instances, and failover only occurs when there are no healthy instances available in the most local set. You should only follow these instructions when every service within a zone has enough capacity to handle requests from downstream services within the same zone.
+!> **Warning**: Locality-aware routing is an advanced feature that may adversely impact service capacity if used incorrectly. By default, this feature routes 100% of traffic to the most local set of service instances, and failover only occurs when there are no healthy instances available in the most local set. You should only follow these instructions when each set of upstream instances (within the same zone, within the same region, and non-local) have enough capacity to handle requests from downstream service instances in their respective zone(s).
```

Comment:
```suggestion
Locality-aware routing is an advanced feature that may adversely impact service capacity if used incorrectly. By default, Consul routes all traffic to the nearest set of service instances and only fails over when no healthy instances are available in the nearest set. You should only follow these instructions when each set of external upstream instances within the same zone and region have enough capacity to handle requests from downstream service instances in their respective zones.
```
There should only be one callout box per section. We don't have a Requirements section on this page, but this is where I would put this information because we are telling people that "enough capacity to handle requests . . ." is a requirement. Even there, this does not rise to the level of needing a red warning callout, which is reserved for warning people about potentially causing catastrophic issues.

Reply:
Ok. It's possible that improperly using this feature would cause existing service instance capacity to become insufficient for handling traffic - immediately or during failover. That could cause partial or total outages in the affected services, which seemed worth calling out. If that doesn't quite meet the bar, I can remove the warning callout.

> We don't have a Requirements section on this page, but this is where I would put this information because we are telling people that "enough capacity to handle requests . . ." is a requirement

Should we add a requirements section, and if so would that just be adding a heading at the current doc position?

Reply:
I moved this below `Introduction` and added a `Requirements` heading - let me know if that looks correct to you